builtin/logical/aws/backend_test.go

```diff
@@ -785,16 +800,25 @@ func testAccStepReadArnPolicy(t *testing.T, name string, value string) logicalte
 	}
 }
 
-func testAccStepWriteArnRoleRef(t *testing.T, roleName string) logicaltest.TestStep {
+func testAccStepWriteArnRoleRef(t *testing.T, vaultRoleName, awsRoleName, awsAccountID string) logicaltest.TestStep {
 	return logicaltest.TestStep{
 		Operation: logical.UpdateOperation,
-		Path:      "roles/" + roleName,
+		Path:      "roles/" + vaultRoleName,
 		Data: map[string]interface{}{
-			"arn": fmt.Sprintf("arn:aws:iam::%s:role/%s", os.Getenv("AWS_ACCOUNT_ID"), roleName),
+			"arn": fmt.Sprintf("arn:aws:iam::%s:role/%s", awsAccountID, awsRoleName),
 		},
 	}
 }
 
+func generateUniqueRoleName(prefix string) string {
```

Comment:
We have an existing function similar to this [RandomWithPrefix](https://github.com/hashicorp/vault/blob/e7c4158e603e96f9ff167e982de017a576938911/helper/testhelpers/testhelpers.go#L80), could you consider using that instead?

Reply:
Yep, thanks for the suggestion! Done.